backend/src/oe/api/op-energy.service.ts

```diff
@@ -68,6 +73,25 @@ export class OpEnergyApiService {
       'value': num,
     };
   }
+
+  public verifyAccountSecret( rawString: string): AccountSecret {
+    if( rawString.length !== 64) {
+      throw new Error('verifyAccountSecret: length');
+    }
+    if( rawString[10] !== '0'
```

Comment:
We should use enum for static values like '0', 'e'  etc. If possible please move this to enum

Reply:
so I am trying to follow your idea and: enum can contain only values of simple types, so I can't keep, for example value of type [number, string] to be able to redefine both position and characters at such positions.

So instead I had refactored to definition of 2 arrays of type [number,string][] such that now we can keep value [10, '0'] and use forEach/reduce functions in creation/validation operations.